zsh/.zshrc

```diff
@@ -193,6 +193,21 @@ function zsh-quickstart-enable-omz-plugins() {
   _zqs-trigger-init-rebuild
 }
 
+function zsh-quickstart-enable-ssh-askpass-require() {
+  _zqs-set-setting ssh-askpass-require true
+  echo "Setting SSH_ASKPASS_REQUIRE=never, this will prompt for your ssh passphrase at the command line"
+  export SSH_ASKPASS_REQUIRE=never
+  echo
+}
+
+function zsh-quickstart-disable-ssh-askpass-require(){
+  _zqs-set-setting ssh-askpass-require false
+  if ! can_haz ssh-askpass; then
+    echo
+    echo "You need to install a SSH askpass gui application for the quickstat to prompt for your ssh key passphrase"
+  fi
```

Comment:
This might be redundant here.

Reply:
I don't think we need it for the disable since the user won't be prompted by `ssh-askpass` in that case.

For the enable, you should check for it and only set `ssh-askpass-require` when `ssh-askpass` is present so they don't get an error every time they `ssh`.

Reply:
Removed